src/NHibernate.Test/NHSpecificTest/NH1521/Fixture.cs

```diff
@@ -11,7 +11,8 @@ public class Fixture
 	{
 		private static void CheckDialect(Configuration configuration)
 		{
-			if (!configuration.Properties[Environment.Dialect].Contains("MsSql"))
+			var dialect = configuration.Properties[Environment.Dialect];
+			if (!dialect.Contains("MsSql") || dialect.Contains("MsSqlCe"))
```

Comment:
`Dialect` is not available here. This is a dialect specific test (I think it's MsSql2005/8Dialect), but unfortunately MsSqlCe dialects starts from the same string :)